src/ui/public/vis/response_handlers/basic.js

```diff
@@ -18,17 +18,18 @@
  */
 
 import { AggResponseIndexProvider } from '../../agg_response';
-import { TabifyTable } from '../../agg_response/tabify/_table';
-
+import { TableResponseHandlerProvider } from './table';
 import { VisResponseHandlersRegistryProvider } from '../../registry/vis_response_handlers';
 
 const BasicResponseHandlerProvider = function (Private) {
```

Comment:
Can we rename the response handler where it's registered further down to `vislib` so people are not trying to use this outside of vislib anymore?

Reply:
or i'll do the same as with table, just won't register it